### ISSUE TITLE
Panic on genji version

### ISSUE THREAD

**Issue by tdemin** [genjidb/genji#261]
After having run `go get -u -v github.com/genjidb/genji/cmd/genji`, trying to run `genji version` results in this output:

```
~ > genji version
Using z.Allocator with starting ref: c74a000000000000
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x919018]

goroutine 1 [running]:
main.main.func2(0xc00007cb80, 0x1, 0x1)
        /home/tdemin/go/src/github.com/genjidb/genji/cmd/genji/main.go:100 +0x58
github.com/urfave/cli.(*Command).Run(0xc0000f6d80, 0xc00007cac0, 0x0, 0x0)
        /home/tdemin/go/src/github.com/urfave/cli/command.go:163 +0x4ed
github.com/urfave/cli.(*App).RunContext(0xc000001980, 0xb69e00, 0xc0000281f0, 0xc00000e080, 0x2, 0x2, 0x0, 0x0)
        /home/tdemin/go/src/github.com/urfave/cli/app.go:306 +0x81f
github.com/urfave/cli.(*App).Run(...)
        /home/tdemin/go/src/github.com/urfave/cli/app.go:217
main.main()
        /home/tdemin/go/src/github.com/genjidb/genji/cmd/genji/main.go:143 +0x612
```

**Comment by tdemin** [comment]
The Git commit I built Genji from is 375ed7aafa0f1d508d14671adb450ac6eb4789dd.

**Comment by asdine** [comment]
I supposed you run `go get` outside of a module? When that happens `debug.ReadBuildInfo` returns nil, which then panics here: https://github.com/genjidb/genji/blob/375ed7aafa0f1d508d14671adb450ac6eb4789dd/cmd/genji/main.go#L100

**Comment by tie** [comment]
I think we should also provide official binaries via GitHub releases. The GOPATH `go get` is essentially development version, so we can’t do much about it.

**Comment by asdine** [comment]
Closed by #262 

@tie good idea, I added an issue about that here https://github.com/genjidb/genji/issues/269
